### ISSUE TITLE
License mislabeled for `js-object.entries`

### ISSUE THREAD

**Issue by brettz9** [davglass/license-checker#226]
Though I've filed https://github.com/m59peacemaker/js-object.entries/issues/2 to add to have `license` added to `package.json`, I would not have expected the README URL to be included in the license type (i.e., "Custom: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global") as there is a bundled CC0-1.0 license.

**Comment by ahamid** [comment]
I agree, it seems rather inaccurate to arbitrarily take the first string that looks like a URL from, e.g., the README file. Some of the versions for my packages with UNKNOWN license are now a pipeline build status svg url.

https://github.com/davglass/license-checker/blob/de6e9a42513aa38a58efc6b202ee5281ed61f486/lib/license.js#L76
https://github.com/davglass/license-checker/blob/de6e9a42513aa38a58efc6b202ee5281ed61f486/lib/index.js#L171
